requirements.txt

```diff
@@ -0,0 +1,2 @@
+Faker==0.9.0
+-e git+git@github.com:bosswissam/pysize.git#egg=pysize
```

Comment:
Does it work? I don't remember whether colon should be be replaced with a slash. (it's true for ssh url though)

Reply:
Yes, it works.
If I to remove all pip packages like https://stackoverflow.com/questions/11248073/what-is-the-easiest-way-to-remove-all-packages-installed-by-pip
And then run
```
pip install -r requirements.txt                    
Collecting Faker==0.9.0 (from -r requirements.txt (line 1))
  Using cached https://files.pythonhosted.org/packages/8d/76/d9e1dde824f059c4c7c62229ac4818d774d451822d84939ffc3376930f0b/Faker-0.9.0-py2.py3-none-any.whl
Obtaining pysize from git+git@github.com:bosswissam/pysize.git#egg=pysize (from -r requirements.txt (line 2))
  Updating /home/sergey_abarovsky/.pyenv/versions/3.5.3/envs/test_venv_project/src/pysize clone
Collecting python-dateutil>=2.4 (from Faker==0.9.0->-r requirements.txt (line 1))
  Using cached https://files.pythonhosted.org/packages/cf/f5/af2b09c957ace60dcfac112b669c45c8c97e32f94aa8b56da4c6d1682825/python_dateutil-2.7.3-py2.py3-none-any.whl
Collecting six>=1.10 (from Faker==0.9.0->-r requirements.txt (line 1))
  Using cached https://files.pythonhosted.org/packages/67/4b/141a581104b1f6397bfa78ac9d43d8ad29a7ca43ea90a2d863fe3056e86a/six-1.11.0-py2.py3-none-any.whl
Collecting text-unidecode==1.2 (from Faker==0.9.0->-r requirements.txt (line 1))
  Using cached https://files.pythonhosted.org/packages/79/42/d717cc2b4520fb09e45b344b1b0b4e81aa672001dd128c180fabc655c341/text_unidecode-1.2-py2.py3-none-any.whl
Installing collected packages: six, python-dateutil, text-unidecode, Faker, pysize
  Found existing installation: pysize 1.0
    Uninstalling pysize-1.0:
      Successfully uninstalled pysize-1.0
  Running setup.py develop for pysize
Successfully installed Faker-0.9.0 pysize python-dateutil-2.7.3 six-1.11.0 text-unidecode-1.2
```
And python code run on my laptop.